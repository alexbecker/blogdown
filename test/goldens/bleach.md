<div class="<img src=x onerror='alert(1)'/>">escaped</div>
<span class="a">allowed</span>
<span onclick="alert('haxx');"></span>
<span class="a" onclick="alert('haxx');"></span>
<img src="x" onerror="alert('haxx');"/>
<span foo="bar"></span>
```a
var x;
```
