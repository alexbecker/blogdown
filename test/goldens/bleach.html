&lt;div&gt;escaped&lt;/div&gt;
<span class="a">allowed</span>
<span></span>
<span class="a"></span>
&lt;img/&gt;
<span></span>
<pre><code class="a">var x;
</code></pre>
